src/zeit/migrate/migrate.py

```diff
@@ -0,0 +1,159 @@
+import UserDict
+import argparse
+import contextlib
+import lxml.etree
+import lxml.objectify
+import re
+import tinydav
+import urlparse
+
+NAMESPACE = 'http://namespaces.zeit.de/CMS/document'
```

Comment:
Das ist der _document_-Namespace. vivi hat mind. noch in Verwendung: workflow, meta, print, tagging.
Evtl bin ich mir daher nicht sicher, wieviel uns mit dieser einen Hilfskonstante hier gedient ist. ;)